Bit Set/BitSet.swift

```diff
@@ -221,6 +221,51 @@ prefix public func ~ (rhs: BitSet) -> BitSet {
   return out
 }
 
+// MARK: - Bit shift operations
+
+/*
+ Note: For bitshift operations, the assumption is that any bits that are
+ shifted off the end of the end of the declared size are not still set.
+ In other words, we are maintaining the original number of bits.
+ */
+
+public func << (lhs: BitSet, numBitsLeft: Int) -> BitSet {
```

Comment:
Looks like you already implemented this method above. Is there a reason for the duplication?

Reply:
For both of these, I'm updating the source that's in the playground as well as the main source file. I couldn't find an obvious way these two files were linked (if they are at all) so it seemed like I needed to update both source files to match.
Unless I'm missing something about how playgrounds work?

Reply:
Hmm yea, that's a good point. Would you mind deleting the entire file? (`Bit Set/BitSet.swift`) Ideally, we'd want only a single set of files with all the source code

Reply:
Is that the new pattern? There are a lot of instances in this repository where the code is duplicated in the playground either as an inclusion or just the code copy-pasted into the playground source file.

I can remove the file outside the playground if you like :)

Reply:
Yea was looking around an noticed this too. I'll start pointing it out more as people update the repo. Even if the code is copy-pasted its a super easy thing to forget to update. Much easier to just have one source of truth.

That'd be great thanks :)